server/sonar-web/src/main/js/apps/issues/workspace-header-view.js

```diff
@@ -105,7 +105,7 @@ export default WorkspaceHeaderView.extend({
 
   bulkChange () {
     const query = this.options.app.controller.getQuery('&', true);
-    const url = '/issues/bulk_change_form?' + query;
+    const url = window.baseUrl + '/issues/bulk_change_form?' + query;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)